recipes/dcv_config.rb

```diff
@@ -22,7 +22,7 @@
 def allow_gpu_acceleration
   # Turn off X
   execute "Turn off X" do
-    command "systemctl set-default multi-user.target"
+    command "systemctl isolate multi-user.target"
```

Comment:
I think it maybe best to include both `systemctl set-default multi-user.target` and `systemctl isolate multi-user.target` according to this guide: https://docs.aws.amazon.com/dcv/latest/adminguide/setting-up-installing-linux-prereq.html#rhel-7.x,-centos-7.x,-amazon-linux-2,-ubuntu-18.x,-and-suse-linux-enterprise-12.x

Reply:
The only call to `set-default` I can find on that guide is the one that sets it to `graphical.target` which is done later in this function after DCV GL is installed.

Reply:
By doing `systemctl isolate` on `multi-user.target` we are turning off X for `graphical.target`, which we turn on after setting up DCV GL later as Tim mentioned.